AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -22,6 +22,9 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+townwork.net##div[class*="mp-app-bnr-"]
+townwork.net##div[class*="AppBanner"]
```

Comment:
better `div[class$="AppBanner"]` (`$` => ending)
and why not `div[class^="mp-app-bnr-"]` (with `^`)